WTWSMS/decisions/LI_target_decisions.txt

```diff
@@ -1,4 +1,4 @@
-targetted_decisions = {	
+decisions = {	
```

Comment:
I'm not responsible for your forum account though. The decision was justified and eventually you will be back. That is how it is.